packages/checkout/sdk-sample-app/src/components/SmartCheckoutForm.tsx

```diff
@@ -187,7 +187,7 @@ export const SmartCheckoutForm = ({ checkout, provider }: SmartCheckoutProps) =>
     updateItemRequirements({
       type: ItemType.ERC20,
       amount,
-      tokenAddress,
+      contractAddress: tokenAddress,
```

Comment:
Should `ERC20ItemRequirement` have token address instead of contract address as well if we are going to make the type change to other smart checkout types?

Reply:
Yep good call, I think it should. I've refactored all ERC20 elements to use tokenAddress instead of contractAddress.